torch/testing/_internal/common_methods_invocations.py

```diff
@@ -111,7 +111,8 @@ def __init__(self,
                  sample_inputs_func=None,  # function to generate sample inputs
                  aten_name=None,  # name of the corresponding aten:: operator
                  variant_test_name='',  # additional string to include in the test name
-                 supports_sparse=False  # supported for sparse
+                 supports_sparse=False,  # supported for sparse
+                 is_linear_map_sparse=False  # supports in-place variant for Sparse Tensors
```

Comment:
this metadata can just be "is_linear_map" since the operation is a linear map for both sparse and non-sparse inputs. The comment should also reflect that this metadata indicates whether the operation is a linear map or not (to be a linear map the operator must be additive and have homogeneity of degree one).

Reply:
@mruberry - made the changes as suggested